container/debian8/builders/rbe-debian8/toolchain_build.bzl

```diff
@@ -0,0 +1,61 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load(
+    "//container/rules:docker_toolchains.bzl",
+    "toolchain_container",
+)
+load("@io_bazel_rules_docker//contrib:test.bzl", "container_test")
+
+def toolchain_build(name, installables_tar = None):
```

Comment:
This macro is hardcoding all the info for the debian8 image and cannot be reused effectively for anything else. iiuc what you are trying to do, is to enable someone to extend the existing toolchain target, by just adding an installables tar. I think we should allow this, but this approach is not the correct one imo. Lets sync up to try to find a better approach.